src/main/resources/assets/buildinggadgets/lang/en_us.lang

```diff
@@ -40,6 +40,7 @@ key.categories.buildingGadgets=Building Gadgets
 key.modeSwitch=Mode Switch
 key.rangeChange=Range
 key.undoKey=Undo/Fuzzy
+key.tooltip=Tooltip Modifier
```

Comment:
Could we maybe change that name? It is very likely to produce conflicts, I think...

(just doing something like buildinggadgets.key.tooltip or some more precise key should work...)

Reply:
Everything in the lang file should have the mod id in it.